generative-ai/chatgpt/prompt-techniques/zero-shot-prompting.md

```diff
@@ -37,9 +37,12 @@ and dialogue generation:
 ---
 ## Practice
 
-With Zero-shot prompting, you ??? to give additional data because the ai ??? trained on a large corpus of text.
+True or False:
+
+With Zero-shot prompting, you do not have to give additional data because the ai was trained on a large corpus of text.
+
+???
+
+- True
+- False
```

Comment:
I feel like we can remove the "True or False:" at the top since the only 2 answers a user will be offered are "True" and "False":

```suggestion
With Zero-shot prompting, you do not have to give additional data because the AI was trained on a large corpus of text.

???

- True
- False
```